index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV - Marharyta Krasikava</title>
+    <link href="style.css" type="text/css" rel="stylesheet">
+</head>
+
+<body class="wrapper">
+    <header class="header">
+        <h1>Marharyta Krasikava</h1>
+        <h3>Junior Front-end Developer</h3>
+        <div class="summary">
+            <p>I'm going to get a new job and be competent and useful of my workplace. My aim is to have an interesting,
+                developing and well-paid occupation. I think I have all the signs of a smart, fast learning, critical
+                thinking and sociable person to achieve a high result.</p>
+        </div>
+    </header>
+    <section class="leftcolumn">
+        <img src="photo3-1.jpg" alt="photo" height="300px" width="300px">
+        <div>
+            <h4>Contact:</h4>
+            <div class="line"></div>
+            <ul>
+                <li><strong>ADRESS:</strong><br>Chepic St., 7-336, 222852, Druzhny, Pukhovichy District, Minsk Region,
+                    Belarus.</li>
+                <li><strong>PHONE:</strong><br><a href="tel:+375291303426">+37529-1303426</a></li>
+                <li><strong>E-MAIL:</strong><br><a
+                        href="mailto:marharytakrasikava@gmail.com">marharytakrasikava@gmail.com</a></li>
+                <li><strong>FACEBOOK: </strong><br><a
+                        href="https://facebook.com/marharyta.krasikava">facebook.com/marharyta.krasikava</a></li>
+            </ul>
+            <h4>Technical skills</h4>
+            <div class="line"></div>
+            <ul>
+                <li>HTML/CSS</li>
+                <div class="progress1"></div>
+                <li>JavaScript</li>
+                <div class="progress2"></div>
+            </ul>
+        </div>
+    </section>
+    <section class="rightcolumn">
+        <h4>Education:</h4>
+        <div class="line-right"></div>
+        <p><strong>2009-2014 - Belarusian State University Of Informatics and Radioelectronics</strong><br><em>Faculty Of Information Technologies And Control</em><br>Specialty "Industrial electronics"</p>
+        <h4>Work experience:</h4>
+        <div class="line-right"></div>
+        <p><strong>- RUP "Minskenergo", subsidiary "TEC-5" (Electrical power station)</strong><br><em>Position:</em> Engineer on electronics</em></p>
+        <h4>Code exapmle:</h4>
+        <div>
+            <input type="checkbox" id="spoiler">
+				<label for="spoiler"> + A Function that implements the Sieve of Eratosthenes:</label>
```

Comment:
not bad idea, but it doesn't look like something, that you can click on. "pointer" cursor isn't enough